tools/topology/topology2/include/pipelines/cavs/google-rtc-aec-capture.conf

```diff
@@ -0,0 +1,112 @@
+#
+# cAVS google-rtc-aec capture pipeline
+#
+# A google-rtc-aec capture pipeline. All attributes defined herein are namespaced
+# by alsatplg to "Object.Pipeline.google-rtc-aec-capture.N.attribute_name"
+#
+# Usage: google-rtc-aec-capture pipeline object can be instantiated as:
+#
+# Object.Pipeline.google-rtc-aec-capture."N" {
+# 	format		"s16le"
+# 	period		1000
+# 	time_domain	"timer"
+# 	channels	2
+# 	rate		48000
+# }
+#
+# Where N is the unique pipeline ID within the same alsaconf node.
+#
+
+<include/components/copier.conf>
+<include/components/pipeline.conf>
+<include/common/audio_format.conf>
+<include/components/google-rtc-aec.conf>
+<include/controls/bytes.conf>
+
+Class.Pipeline."google-rtc-aec-capture" {
+
+	DefineAttribute."index" {}
+
+	<include/pipelines/pipeline-common.conf>
+
+	attributes {
+		!constructor [
+			"index"
+		]
+
+		!immutable [
+			"direction"
+		]
+
+		#
+		# google-rtc-aec-capture objects instantiated within the same alsaconf node must have
+		# unique pipeline_id attribute
+		#
+		unique	"instance"
+	}
+
+	Object.Widget {
+		copier."1" {
+			copier_type	"host"
+			type	"aif_out"
+			num_audio_formats 2
+
+			node_type $HDA_HOST_INPUT_CLASS
+		}
+
+		pipeline."1" {
+			priority	0
+			lp_mode		0
+		}
+
+		google-rtc-aec."1" {
+			Object.Control.bytes."1" {
+				name google-rtc-aec_blob
+				access  [
+					tlv_read
+					tlv_callback
+				]
+
+				Object.Base.extops."extctl" {
+					get 258
+					put 0
+				}
+				max 4096
+				Object.Base.data."google-rtc-aec_blob" {
+					bytes "
```

Comment:
any chance this can be included for cleanliness?

Reply:
sure.